coding/endianness.hpp

```diff
@@ -8,7 +8,15 @@
 
 // #define ENDIAN_IS_BIG
 
-inline bool IsBigEndian()
+// @TODO(bykoianko) This method returns false since 05.12.2010. That means only little endian
+// architecture is supported. When it's necessary to support a big endian system:
+// * method IsBigEndianMacroBased() should be implement based on IsLittleEndian() method
+// * method SwapIfBigEndianMacroBased() should be implemented based on IsLittleEndian() method
+// * all serialization and deserialization of rs_bit_vector and the other rs-structures
+//   should be implemented taking into account endianness
+// * variants should be implemented taking into account endianness
```

Comment:
What do you mean here?

Reply:
I mean functions implemented in coding/variant.hpp. I've fixed the comment:
`// * methods in coding/varint.hpp should be implemented taking endianness into account`

Reply:
Yes. You're right. I'm removing this line.